typescript/phoenix-sdk/examples/deserializeClock.ts

```diff
@@ -3,10 +3,10 @@ import { BN } from "bn.js";
 
 import * as Phoenix from "../src";
 
-// Ex: ts-node tests/market.ts
-export async function market() {
-  const connection = new Connection("https://api.mainnet-beta.solana.com");
-  const phoenix = await Phoenix.Client.create(connection, "mainnet");
+// Ex: ts-node examples/deserializeClock.ts
+export async function deserializeClock() {
+  const connection = new Connection("https://api.devnet.solana.com");
```

Comment:
What's wrong with getting the mainnet clock? FWIW I think the public mainnet RPC is slightly more reliable than the public devnet RPC

Reply:
was testing client creation w/ devnet, reverted.